docs/kratos/concepts/security.mdx

```diff
@@ -73,16 +73,59 @@ Ory OAuth2 and OpenID Connect is a certified OAuth2 and OpenID Connect provider.
 
 ## CAPTCHAs
 
-:::info
-
-Captcha protection is being tested and is not yet available for general use. If you are interested, please
-[contact us](https://ory.sh/contact).
+Ory Identities supports protecting the registration and login endpoints with CAPTCHA challenges. This is useful to prevent
+credential stuffing, brute force and other automated attacks.
 
-:::
+#### Prerequisites
 
-Ory Identities supports protecting the registration and login endpoints with captcha challenges. This is useful to prevent
-credential stuffing, brute force and other automated attacks.
+Before proceeding, ensure you are on a plan that supports this feature. If you need CAPTCHA support, please
+[contact us](https://www.ory.sh/contact/).
 
-Supported captcha providers are:
+Supported CAPTCHA providers are:
 
 - [Cloudflare Turnstile](https://developers.cloudflare.com/turnstile)
+
+```mdx-code-block
+<Tabs groupId="console-or-api">
+<TabItem value="console" label="Ory Console">
+```
+
+1. Go to <ConsoleLink route="project.authentication" />.
+2. Toggle **CAPTCHA protection**.
+3. Click **Save**.
+
+4. Navigate to your registration or login screen to test the CAPTCHA protection.
+
+```mdx-code-block
+</TabItem>
+<TabItem value="cli" label="Ory CLI">
+```
+
+```shell
+ory patch identity-config --project <project-id> --workspace <workspace-id> \
+  --replace '/selfservice/mehods/captcha/enabled=true'
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="api" label="API">
+```
+
+```shell
+curl -X PATCH --location "https://api.console.ory.sh/projects/$PROJECT_ID" \
+     -H "Authorization: Bearer $WORKSPACE_API_KEY" \
+     -H "Content-Type: application/json" \
+     -d '[
+           {
+             "op": "replace",
+             "path": "/services/identity/config/selfservice/methods/captcha/enabled",
+             "value": true
+           }
+         ]' \
+   | jq ".project.services.identity.config.selfservice.methods.captcha"
+```
+
+```mdx-code-block
+</TabItem>
```

Comment:
I would rather not have this, as this API is not really stable and documented. Leave Console & CLI for now.